lib/css/components/select.less

```diff
@@ -0,0 +1,194 @@
+        --_se-select-fs: var(--fs-headline1);
+        --_se-select-px: 0.5em;
+        --_se-select-py: 0.4em;
+    }
+
+    .has-error & {
+        --_se-select-bc: var(--red-400);
+        --_se-select-bc-focus: var(--red-400);
+        --_se-select-bs-focus: 0 0 0 var(--su-static4) var(--focus-ring-error);
+    }
+
+    .has-success & {
+        --_se-select-bc: var(--green-400);
+        --_se-select-bc-focus: var(--green-400);
+        --_se-select-bs-focus: 0 0 0 var(--su-static4) var(--focus-ring-success);
+    }
+
+    .has-warning & {
+        --_se-select-bc: var(--yellow-600);
+        --_se-select-bc-focus: var(--yellow-600);
+        --_se-select-bs-focus: 0 0 0 var(--su-static4) var(--focus-ring-warning);
+    }
+
+    // CHILD ELEMENTS
+    &,
+    & > select {
+        &::-webkit-contacts-auto-fill-button {
+            background-color: var(--black); // In Safari, make the autocomplete button darkmode-aware
+        }
+
+        &:-webkit-autofill {
+            border-color: var(--blue-300);
+            -webkit-text-fill-color: var(--black);
+            -webkit-box-shadow: 0 0 0 1000px var(--theme-secondary-050) inset; // This acts as a background color by stretching an inset box shadow across the input
+            transition: background-color 0s 50000s; // A hack to infinitely delay background styles that come from the browser.
+
+            &:focus {
+                border-color: var(--blue-300);
+                // Since the box shadow is overwritten to show a background, we have to re-add the focus outline
+                -webkit-box-shadow: 0 0 0 1000px var(--blue-050) inset, 0 0 0 var(--su-static4) var(--focus-ring);
+            }
+        }
+    }
+
+    &:before,
+    &:after { // menu arrows
+        border-color: var(--_se-arrow-bc);
+        border-style: solid;
+        border-width: var(--_se-arrow-size);
+        content: "";
+        pointer-events: none;
+        position: absolute;
+        right: calc(var(--su-static12) + var(--su-static1));
+        z-index: var(--zi-selected);
+    }
+
+    &:after {
+        border-bottom-width: 0;
+        top: calc(50% + var(--su-static1));
+    }
+
+    &:before {
+        border-top-width: 0;
+        top: calc(50% - calc(var(--_se-arrow-size) + var(--su-static1)));
+    }
+
+    > select { // Menu
+        // CONTEXTUAL STYLES
+        fieldset[disabled] &,
+        &[disabled] {
+            cursor: not-allowed;
+            opacity: var(--_o-disabled-static);
+        }
+
+        &[readonly],
+        .is-readonly & { // [1]
+            --_se-select-bc: var(--bc-light);
+            --_se-select-bg: var(--black-050);
+            --_se-select-fc: var(--black-200);
+
+            .highcontrast-mode({
+                --_se-select-fc: var(--fc-light);
+            });
+
+            cursor: not-allowed;
+        }
+
+        // CHILD ELEMENTS
+        &::-moz-focus-inner {
+            outline: none !important;
+        }
+
+        &:-moz-focusring {
```

Comment:
PR Note - double check, necessary or is this on a standards track now?

Reply:
AFAICT, we're good to dump this. I tested and saw no difference in FF with or without this rule.

![image](https://user-images.githubusercontent.com/647177/215874873-9f470dea-ef60-44f8-b13a-377befbe644b.png)

Furthermore, [MDN says](https://developer.mozilla.org/en-US/docs/Web/CSS/:-moz-focusring):
> Non-standard: This feature is non-standard and is not on a standards track. Do not use it on production sites facing the Web: it will not work for every user. There may also be large incompatibilities between implementations and the behavior may change in the future.